src/Sylius/Bundle/TaxationBundle/Resources/translations/validators.pl.yml

```diff
@@ -16,6 +16,7 @@ sylius:
             not_blank: Proszę podać nazwę stawki podatkowej.
         amount:
             not_blank: Proszę podać wartość stawki podatkowej.
+            invalid: Wartość stawki podatkowej jest nieprawidłowa.
```

Comment:
you should add translation only for English, the rest of it is translate by `crowdIn`